libs/boost/Makefile

```diff
@@ -16,15 +16,15 @@ include $(INCLUDE_DIR)/nls.mk
 include $(INCLUDE_DIR)/target.mk 
 
 PKG_NAME:=boost
-PKG_VERSION:=1.64.0
-PKG_SOURCE_VERSION:=1_64_0
+PKG_VERSION:=1.65.0
+PKG_SOURCE_VERSION:=1_65_0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)_$(PKG_SOURCE_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://sourceforge.net/projects/boost/files/boost/$(PKG_VERSION)
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)_$(PKG_SOURCE_VERSION)
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)_$(PKG_SOURCE_VERSION)
-PKG_MD5SUM:=7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
+PKG_MD5SUM:=ea26712742e2fb079c2a566a31f3266973b76e38222b9f88b387e3c8b2f9902c
```

Comment:
According to Travis CI
```WARNING: Use of PKG_MD5SUM is deprecated, switch to PKG_HASH```

Reply:
@thess 
Thanks!
One question. Should I pay attention to the Travis failure? Is it working properly?

Reply:
Hi @ClaymorePT 
The only "normal" failure is when travis timeout (we only have 2 vcpus for 60 mins, sometimes it's not enough to rebuild all dependencies)
Else everytime Travis is not working properly don't hesitate to ping me

Reply:
@champtar 

Ok. Thank you for the warning. For now, it is still building the package. I will wait.